locust/web.py

```diff
@@ -255,10 +260,50 @@ def ramp():
     response.headers["Content-type"] = "application/json"
     return response
 
+@app.route("/inputCSV", methods=["POST"])
+def inputCSV():
+    print(request.method)
+    assert request.method == "POST"
+
+    csvfile = request.files['csv_file']
+    if not csvfile:
+        return "No file"
```

Comment:
make the return of the web service as json

Reply:
no function read it, only warning like stats/reset function